src/lib/components/select/QSelect.svelte

```diff
@@ -102,7 +102,7 @@
       return;
     }
 
-    value = optionValue;
+    (value as string) = optionValue;
```

Comment:
That's how the errors were removed? I don't feel like we should merge it like this.

Reply:
I got confirmation on my issue on `svelte-tools` repo that this problem comes from them and that it's pending release.
It's temporary